packages/file_selector/file_selector_windows/windows/test/test_utils.h

```diff
@@ -0,0 +1,91 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#ifndef PACKAGES_FILE_SELECTOR_FILE_SELECTOR_WINDOWS_WINDOWS_TEST_TEST_UTILS_H_
+#define PACKAGES_FILE_SELECTOR_FILE_SELECTOR_WINDOWS_WINDOWS_TEST_TEST_UTILS_H_
+
+#include <comdef.h>
+#include <comip.h>
+#include <shlobj.h>
+#include <shobjidl.h>
+#include <windows.h>
+
+#include <memory>
+#include <string>
+#include <type_traits>
+#include <variant>
+
+#include "file_dialog_controller.h"
+
+_COM_SMARTPTR_TYPEDEF(IShellItem, IID_IShellItem);
+_COM_SMARTPTR_TYPEDEF(IShellItemArray, IID_IShellItemArray);
+
+namespace file_selector_windows {
+namespace test {
+
+// Creates a temp file, managed as an IShellItem, which will be deleted when
+// the instance goes out of scope.
+//
+// This creates a file on the filesystem since creating IShellItem instances for
+// files that don't exist is not-trivial.
```

Comment:
pedantic nit: non-trivial or not trivial :P

Reply:
Yep, that was a typo. Fixed.